src/test/java/praktikum/OrderFormTest.java

```diff
@@ -0,0 +1,82 @@
+package praktikum;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import praktikum.pageObjects.AboutRentPage;
+import praktikum.pageObjects.MainPage;
+import praktikum.pageObjects.OrderFormPage;
+import praktikum.pageObjects.PopUpWindow;
+
+import static org.junit.Assert.assertTrue;
+import static praktikum.pageObjects.constant.OrderButton.HEADER_BUTTON;
+import static praktikum.pageObjects.constant.OrderButton.MIDDLE_BUTTON;
+
+@RunWith(Parameterized.class)
+public class OrderFormTest extends BrowserSettings {
+    private final Enum button;
+    private final String firstName;
+    private final String lastName;
+    private final String address;
+    private final int stateMetroNumber;
+    private final String phoneNumber;
+    private final String date;
+    private final String comment;
+
+
+    public OrderFormTest(Enum button, String firstName, String lastName, String address, int stateMetroNumber, String phoneNumber,
+                         String date, String comment) {
+        this.button = button;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.address = address;
+        this.stateMetroNumber = stateMetroNumber;
+        this.phoneNumber = phoneNumber;
+        this.date = date;
+        this.comment = comment;
+    }
+
+    @Parameterized.Parameters
+    public static Object[][] getParameters() {
+        return new Object[][]{
+                {HEADER_BUTTON, "Имя", "Фамилия", "Адрес 1", 123, "76665554433", "28.05.2024", "комментарий"},
+                {HEADER_BUTTON, "Иван", "Иванов", "Адресок 2", 12, "76665554433", "28.06.2024", "славного денёчка"},
+                {MIDDLE_BUTTON, "Полина", "Петрова", "Жилище 3", 1, "76665554433", "28.07.2024", "хорошего настроения"},
```

Comment:
Кейсы для MIDDLE_BUTTON падают на моменте нажатия кнопки с ошибкой:

org.openqa.selenium.ElementClickInterceptedException: Element <button class="Button_Button__ra12g Button_Middle__1CSJM"> is not clickable at point (612,46) because another element <div class="Header_Header__214zg"> obscures it

Reply:
Не получается получить эту ошибку. 
Все тесты в IntelliJ IDEA на моменте нажатия кнопки (MIDDLE_BUTTON) проходят успешно для обоих браузеров.

![воспроизведение теста OrderFormTest](https://github.com/user-attachments/assets/2cc19c7b-0512-4f49-b02d-b67bbae7f747)